src/components/home/About.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Title from "@/components/Title";
+
+import Image from "next/image";
+import cloud from "@/public/assets/cloud.svg";
+import navlogo from "@/public/images/navlogo.webp";
+
+const About = () => {
+  return (
+    <div className="flex flex-col items-center">
+      <div className="flex items-center px-32 z-1">
```

Comment:
![image](https://github.com/acm-ucr/isa-website/assets/91030482/fc60743a-0c4f-42f3-9bf0-4d60db6cec8b)
logo should be closer to the texts